test/src/test/java/jenkins/triggers/ReverseBuildTriggerTest.java

```diff
@@ -129,10 +118,7 @@
         r.waitUntilNoActivity();
         assertEquals(1, downstream.getLastBuild().number);
         // Alice can see upstream, so downstream gets built, but the upstream build cannot see downstream:
-        perms = new HashMap<Permission,Set<String>>();
-        perms.put(Item.READ, new HashSet<String>(Arrays.asList("alice", "bob")));
-        upstream.removeProperty(AuthorizationMatrixProperty.class);
-        upstream.addProperty(new AuthorizationMatrixProperty(perms));
+        auth.grant(Item.READ).onItems(upstream).to("alice", "bob");
```

Comment:
`bob` already had it so would suffice to only add the grant for `alice`, but this is fine too.